packages/atomic/src/components/atomic-result-template/atomic-result-template.tsx

```diff
@@ -116,7 +118,37 @@ export class AtomicResultTemplate {
   }
 
   private getContent() {
-    return this.getTemplateElement().content;
+    // TODO: reorganise
+    const childTemplatesBySection: HTMLAtomicResultChildrenTemplateElement[][] =
+      [];
+    const content = this.getTemplateElement().content;
+    const childrenTemplates = content.querySelectorAll(
+      'atomic-result-children-template'
+    );
+    childrenTemplates.forEach((template) => {
```

Comment:
this might have to change to
```js
    const childrenTemplates = Array.from(
      content.querySelectorAll('atomic-result-children-template')
    ).filter((template) => {
      return !elementHasAncestorTag(
        template,
        'atomic-result-children-template'
      );
    });
```
to make sure we don't add nested children templates to be used on this level of children